operations/fetch.go

```diff
@@ -103,14 +103,14 @@ func Fetch() cli.Command {
 				return errors.Wrap(err, "problem loading configuration")
 			}
 
-			_ = conf.GetRestCommunicator(ctx)
+			client := conf.GetRestCommunicator(ctx)
 
 			ac, rc, err := conf.getLegacyClients()
 			if err != nil {
 				return errors.Wrap(err, "problem accessing evergreen service")
 			}
 
-			notifyUserUpdate(ac)
+			notifyUserUpdate(client)
```

Comment:
can't we just move this to the getRestCommunicator method? 

I think there are at least some places where we can drop using the "ac" client all together..